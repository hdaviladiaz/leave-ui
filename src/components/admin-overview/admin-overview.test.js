import React from 'react';
import { shallow } from 'enzyme';
import AdminOverview from './admin-overview';

it('should render AdminOverview without crashing', () => {
  shallow(<AdminOverview />);
});
