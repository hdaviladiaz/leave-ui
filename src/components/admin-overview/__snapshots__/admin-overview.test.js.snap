// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AdminOverview as snapshot 1`] = `
<div
  className="panel panel-default"
  id={undefined}
>
  <div
    className="panel-body"
  >
    <div
      className="admin-overview"
    >
      <div
        className="col-xs-12"
      >
        <h4>
<<<<<<< HEAD
<<<<<<< HEAD
          24 junio 2017
=======
          26 junio 2017
>>>>>>> 29411eb3ab190b16ced5392a502895bedb3c6897
=======
          23 junio 2017
>>>>>>> 3cf583a66981cc9e5894cf6d005bba096a45f1db
        </h4>
        <span
          className="text-muted"
        >
           dias disponibles
        </span>
      </div>
      <div
        className="admin-overview-box-container no-padding col-xs-12"
      >
        <div
          className="admin-overview-box col-md-4"
        >
          <div
            className="admin-overview-box-header"
          >
            Pendientes
          </div>
          <div
            className="admin-overview-box-body"
          >
            <div
              className="admin-overview-box-title"
            />
          </div>
        </div>
        <div
          className="admin-overview-box last admin-overview-box-center col-md-8"
        />
      </div>
    </div>
  </div>
</div>
`;
